.github/workflows/ci.yml

```diff
@@ -55,3 +55,18 @@ jobs:
 
       - name: Typecheck
         run: pnpm typecheck
+
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v5
+
+      - name: Setup
+        uses: ./tooling/github/setup
+
+      - name: Copy env
+        shell: bash
+        run: cp .env.example .env
+
+      - name: Test
+        run: pnpm test
```

Comment:
I would suggest creating a new file instead of altering the existing ci.yml from turbo, fewer changes of conflicts.

I've worked on this in my own fork:

https://github.com/ecoologic/create-t3-turbo/blob/main/.github/workflows/ci-test.yaml

and it also compares coverage.

The setup script is kinda interesting too, though very tough:
https://github.com/ecoologic/create-t3-turbo/tree/main/scripts

Reply:
![what-he-said](https://media1.giphy.com/media/v1.Y2lkPTc5MGI3NjExc2FqeTVsOWZsZjRvaWEzdHRjZzZkNjh1NmV4NGR2MnBzazBteHhjaiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/5QNQv6xmVEaabGsYrg/giphy.gif)